pkg/controller/resourcequota/resource_quota_monitor.go

```diff
@@ -103,19 +103,6 @@ type QuotaMonitor struct {
 	updateFilter UpdateFilter
 }
 
-// NewMonitor creates a new instance of a QuotaMonitor
-func NewMonitor(informersStarted <-chan struct{}, informerFactory informerfactory.InformerFactory, ignoredResources map[schema.GroupResource]struct{}, resyncPeriod controller.ResyncPeriodFunc, replenishmentFunc ReplenishmentFunc, registry quota.Registry) *QuotaMonitor {
```

Comment:
@HirazawaUi This function is part of public API and was consumed at least in https://github.com/openshift/cluster-policy-controller/blob/91dfd166cc6a305dfc93b7e45d01fa31e57fd437/pkg/quota/clusterquotareconciliation/reconciliation_controller.go#L104 to my knowledge. I'm not sure if there are other consumers of it or not. But I'd like to ask for a bit more due diligence before removing public function without prior notice. cc @thockin